### ISSUE TITLE
GangaError missing when trying to raise error after zero DiracFile replicas found

### ISSUE THREAD

**Issue by dhcroft** [ganga-devs/ganga#406]
In ganga  6.1.18 I tried to get a list of replicas for an LFN but the error message failed as GangaErrors was apparently undefined. See partial log file below.

```
$ lb-run ganga ganga
...
*** Welcome to Ganga ***
Version: 6.1.18
...
[10:34:02]
Ganga In [1]: df = DiracFile(lfn="/lhcb/LHCb/Collision15/CALIBRATION.DST/00049575/0000/00049575_00000021_1.calibration.dst")
...
Ganga In [3]: df.getReplicas()
Ganga.GangaDirac.Lib.Files         : ERROR    Couldn't find replicas for: /lhcb/LHCb/Collision15/CALIBRATION.DST/00049575/0000/00049575_00000021_1.calibration.dst
Ganga.Runtime.bootstrap            : ERROR    Error: global name 'GangaError' is not defined
Ganga.Runtime.bootstrap            : ERROR    Unknown/Unexpected ERROR!!
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
/cvmfs/lhcb.cern.ch/lib/lhcb/GANGA/GANGA_v601r18/install/ganga/bin/ganga in <module>()
----> 1 df.getReplicas()

/cvmfs/lhcb.cern.ch/lib/lhcb/GANGA/GANGA_v601r18/install/ganga/python/GangaDirac/Lib/Files/DiracFile.py in getReplicas(self, forceRefresh)
    463                 else:
    464                     logger.error("Couldn't find replicas for: %s" % str(self.lfn))
--> 465                     raise GangaError("Couldn't find replicas for: %s" % str(self.lfn))
    466                 logger.debug("getReplicas: %s" % str(self._storedReplicas))
    467

NameError: global name 'GangaError' is not defined


```

**Comment by egede** [comment]
@dhcroft Clearly a bug. Thanks for reporting.

**Comment by rob-c** [comment]
I think this has likely been fixed in 6.3 as the handling of Errors from Dirac has been revamped. Also GangaError isn't mentioned in the file at all.
